ingestion/functions/common/parsing_lib.py

```diff
@@ -139,6 +141,7 @@ def write_to_server(
         cases_batch_size: int):
     """Upserts the provided cases via the G.h Case API."""
     put_api_url = f"{common_lib.get_source_api_url(env)}/cases/batchUpsert"
+    upload_status_url = f"{common_lib.get_source_api_url(env)}/sources/{source_id}/uploads/{upload_id}"
```

Comment:
Minor, but can DRY out function calls by setting `common_lib.get_source_api_url(env)` to a variable